<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMakePanelClass</class>
 <widget class="QWidget" name="CMakePanelClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMakePanel</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="5">
    <widget class="QRadioButton" name="rBtnX86">
     <property name="text">
      <string>x86</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>CMakelist dir</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="7">
    <widget class="QTextEdit" name="textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Command:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QRadioButton" name="rBtnWin32">
     <property name="text">
      <string>Win 32</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="6">
    <widget class="QLineEdit" name="lEdOutputDir">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Directory where solutions are created&lt;/p&gt;&lt;p&gt;relative to working directory.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Output relative dir</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Compiler</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QRadioButton" name="rBtnX64">
     <property name="text">
      <string>x64</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="toolTip">
      <string>Enable step</string>
     </property>
     <property name="text">
      <string>Enable step</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="6">
    <widget class="QLineEdit" name="lEdCMakeList">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Where CMakelist.txt is located&lt;/p&gt;&lt;p&gt;relative to working directory&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="cboCompiler">
     <property name="toolTip">
      <string>Compiler used</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QCheckBox" name="chBxQuote">
     <property name="text">
      <string>Use quotes for compiler name</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QCheckBox" name="chBxCompilerName">
     <property name="text">
      <string>use compilerName</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
